src/index.js

```diff
@@ -402,6 +403,18 @@ function createConfig(options, entry, format, writeMeta) {
 	const outputDir = dirname(absMain);
 	const outputEntryFileName = basename(absMain);
 
+	const argv1 = process.argv[1];
+	let typescriptLibPath;
+	if (argv1 != null) {
+		// If argv1 exists, try to require 'typescript' from there.
+		typescriptLibPath = resolveFrom(dirname(argv1), 'typescript');
```

Comment:
I think this could be changed to use the cwd rather than assuming microbundle is run from the cwd. It also needs to use `.silent()`, otherwise this throws when there's no local typescript.

```suggestion
		typescriptLibPath = resolveFrom.silent(options.cwd, 'typescript');
```

Reply:
You're right. There's no reason to prevent project's "typescript" to be required by global microbundle.  Applied your simplified solution.